.github/workflows/middleware.yml

```diff
@@ -59,3 +59,22 @@ jobs:
 
       - name: Build
         run: npm run build
+
+  publish:
```

Comment:
do these jobs all run at the same time?

Reply:
Also you not publishing compile code just src

Reply:
Also you cant call npm publish, it will fail if the version is not bumped. I would recommend a GitHub actions app that check to see if there is a different version then it will publish